### PR TITLE
Create getting_started

### PR DESCRIPTION
**Pull request by zrkelastx**
Migration of:
https://elastx.zendesk.com/knowledge/articles/218042523/en-us?brand_id=2236496

### CODE REVIEW

File: content/en/docs/openstack-iaas/guides/getting_started [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+1. Go to "Project" > "Compute" > "Instances" > from the "Actions" menu on the instance you created select "Associate Floating IP"
+> select the "+" button next to the "IP Address" field
+> select "Pool" "elx-public1" > "Allocate IP"
+> "Associate"
+
+## Log in to your new instance 
+Use the floating IP and the ssh key you created.
+1. ssh -i mykeypair.pem ubuntu@1.2.3.4
+In this example the ssh key pair I created was named mykeypair and the public ip is "1.2.3.4" and the image I used was an Ubuntu image.
+If you used any other image you can find the default user names here https://elastx.zendesk.com/hc/en-us/articles/214237986-How-to-log-in-to-your-new-Linux-instance
```
Comment (Doctor-love) [inline content/en/docs/openstack-iaas/guides/getting_started:52]: Maybe we could migrate this page from the support portal as well (as it's a dependency) in the same PR?

File: content/en/docs/openstack-iaas/guides/getting_started [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
```
Comment (Doctor-love) [inline content/en/docs/openstack-iaas/guides/getting_started:53]: Some redundant empty lines

**Comment by Doctor-love** [comment]
@zrkelastx : If you run "test-locally.sh" and check out the formatting of the guide, you'll notice that the syntax in the TODO steps don't quite render as you expect.

File: content/en/docs/openstack-iaas/guides/getting_started (hunk 1 above)
Comment (zrkelastx) [reply content/en/docs/openstack-iaas/guides/getting_started]: yes, I will include it.

**Comment by zrkelastx** [comment]
> @zrkelastx : If you run "test-locally.sh" and check out the formatting of the guide, you'll notice that the syntax in the TODO steps don't quite render as you expect.

thnx, attempted it during last commit.ttt

File: content/en/docs/openstack-iaas/guides/getting_started.md [hunk 3]
```diff
@@ -29,29 +31,43 @@ Now the network is up and ready for you to create the first instance.
...
+Linux flavor	User
+CentOS 6	cloud-user
+CentOS 7	centos
+CoreOS	core
+Fedora	fedora
+Redhat	cloud-user
+Ubuntu	ubuntu
+Debian	debian
+Heat instances	ec2-user
+When using Heat to deploy instances the user name will be ec2-user instead.
```
Comment (Doctor-love) [inline content/en/docs/openstack-iaas/guides/getting_started.md:73]: Perhaps consider using the table syntax instead? Example:
```
| Property | Value |
|-------------|------------------------------------|
| Server    | smtp.elastx.email
| Port        | 587
| Protocol | SMTP + STARTTLS (preferred)
```

In markdown, if you want to force a newline, you'll have to end each line with two spaces.

File: content/en/docs/openstack-iaas/guides/getting_started.md [hunk 4]
```diff
@@ -29,29 +31,43 @@ Now the network is up and ready for you to create the first instance.
...
-> set instance name > "Next"
-> select "Image" in"Select Boot Source" >  select "No" in "Create New Volume"
-> select image (ubuntu-20.04-server-latest for example) > "Next"
-> select a flavor (v1-standard-1 for example) > "Next"
-> your network should already be selected > "Next"
-> you do not need to select any port > "Next"
-> add the security group you created earlier > "Next"
-> the key pair you created earlier should already be selected 
-> "Launch instance"
+1. set instance name > "Next"
```
Comment (Doctor-love) [inline content/en/docs/openstack-iaas/guides/getting_started.md:34]: Fix casing for items.

File: content/en/docs/openstack-iaas/guides/getting_started.md [hunk 5]
```diff
@@ -29,29 +31,43 @@ Now the network is up and ready for you to create the first instance.
...
+1. select "Image" in"Select Boot Source" >  select "No" in "Create New Volume"
```
Comment (Doctor-love) [inline content/en/docs/openstack-iaas/guides/getting_started.md:35]: Add lacking space

**Review by Doctor-love — CHANGES_REQUESTED** [review]

File: content/en/docs/openstack-iaas/guides/getting_started.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+## Create Network
+1. Go to "Project" > "Network" > "Networks"
+1. Select "Create Network" > set a network name > "Next" 
+1. set subnet name and network address (10.0.1.0/24 for example) > "Next" 
+1. add "DNS Name Servers" > "Create"
+
+## Create Router
+1. Go to "Project" > "Network" > "Routers"
+2. Select "Create Router" > set a router name and select the public network "ext-net-01" > "Create Router"
```
Comment (Doctor-love) [inline content/en/docs/openstack-iaas/guides/getting_started.md:19]: Isn't this called "elx-public1" these days?

File: content/en/docs/openstack-iaas/guides/getting_started.md (hunk 3 above)
Comment (zrkelastx) [reply content/en/docs/openstack-iaas/guides/getting_started.md:73]: Good idea.

File: content/en/docs/openstack-iaas/guides/getting_started.md (hunk 4 above)
Comment (zrkelastx) [reply content/en/docs/openstack-iaas/guides/getting_started.md:34]: Fixed loads of casing.

File: content/en/docs/openstack-iaas/guides/getting_started.md (hunk 3 above)
Comment (zrkelastx) [reply content/en/docs/openstack-iaas/guides/getting_started.md]: fixxed.

File: content/en/docs/openstack-iaas/guides/getting_started.md (hunk 6 above)
Comment (zrkelastx) [reply content/en/docs/openstack-iaas/guides/getting_started.md:20]: Nice catch, fixxed too.

File: content/en/docs/openstack-iaas/guides/getting_started.md [hunk 7]
```diff
@@ -55,19 +59,23 @@ In this example the ssh key pair I created was named mykeypair and the public ip
 ```ssh -i mykeypair.pem ubuntu@1.2.3.4```
 
 The username is different depending on the Linux flavor you are using but you will always use the keypair and not a password.
-This is the ssh command you will use is you login from a Linux client. Here is the generic pattern:
+Thiis is the generic pattern to login from a Linux client:
```
Comment (Doctor-love) [inline content/en/docs/openstack-iaas/guides/getting_started.md:62]: "Thiis" -> "This"

**Review by Doctor-love — CHANGES_REQUESTED** [review]
Soon ready for merge - except small spelling issue and the public network name!

File: content/en/docs/openstack-iaas/guides/getting_started.md (hunk 7 above)
Comment (zrkelastx) [reply content/en/docs/openstack-iaas/guides/getting_started.md:62]: iiiiii will handle this one too :))

(sorry about the elx-public1 - that is done on both line 20 and 51 now.)

Thank you for your patience.

**Review by Doctor-love — APPROVED** [review]
Good job!

**Pull request merged**